clients/mobile/src/authentication/services/authErrorHandler.ts

```diff
@@ -0,0 +1,42 @@
+export const handleAuthError = (error: unknown): string => {
+  let errorMessage = 'Unknown error occurred.'
+
+  if (error instanceof Error) {
+    switch (error.name) {
+      case 'NotAuthorizedException':
+        errorMessage = 'Unauthorized to sign out.'
```

Comment:
wrong grammer

Reply:
Vai What is the current one i have asked ChatGPT and used Grammarly to correct it but both say okay.